ext/drsyscall/drsyscall_linux.c

```diff
@@ -964,12 +964,13 @@ handle_semctl(void *drcontext, cls_syscall_t *pt, sysarg_iter_info_t *ii,
     /* int semctl(int semid, int semnum, int cmd, union semun arg) */
```

Comment:
> i#7303: Port DrMemory PR2546.

I found the title confusing: how about `i#7303 drsyscall: Pull in latest drmemory/drsyscall changes`.